mmseg/models/backbones/fast_scnn.py

```diff
@@ -364,7 +387,7 @@ def __init__(self,
             fusion_out_channels,
             conv_cfg=self.conv_cfg,
             norm_cfg=self.norm_cfg,
-            act_cfg=self.act_cfg,
+            dwconv_act_cfg=self.act_cfg,
```

Comment:
act_cfg?

Reply:
`dwconv_act_cfg` is the `act_cfg` in original version